app/db/models/law_policy/family.py

```diff
@@ -17,7 +17,7 @@ class FamilyCategory(BaseModelEnum):
 
     EXECUTIVE = "Executive"
     LEGISLATIVE = "Legislative"
-    UNFCCC = "UNFCCC"
+    UNFCCC = ORGANISATION_UNFCCC
```

Comment:
Hmmmm :thinking: ... maybe this should not be used here.

Reply:
yeah, probably not